src/translations/en.json

```
@@ -292,7 +292,8 @@
         "five": {
           "selectCardToDiscard": "You must select one card to discard",
           "incorrectCard": "Incorrect card played",
-          "fiveDeckIsEmpty": "Cannot resolve 5 one-off with an empty deck"
+          "fiveDeckIsEmpty": "Cannot resolve 5 one-off with an empty deck",
+          "wrongPhase": "You can't resolve a five right now."
```

We have a test to confirm key order across translation files so let's make these consistent with the ones below
```suggestion
          "wrongPhase": "You can't resolve a five right now.",
          "fiveDeckIsEmpty": "Cannot resolve 5 one-off with an empty deck"
```